2-mandatory/2-financial-times.js

```diff
@@ -23,15 +41,30 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let headLineWN = [];
+    let numberArray = ['0','1','2','3','4','5','6','7','8','9'];
+    let numberArray2 = ['0123456789'];
+    for(let element of allArticleTitles){
+        for(char of element){
+            if(char.includes(numberArray)){
+                headLineWN.push(element);
+                break;
+            }
+        }
+    }
+    return headLineWN;
 }
 
```

Comment:
This doesn't pass the related test. The issue with the headlinesWithNumbers function is that the includes() method is being used on the char variable which is a single character, and you are passing an array to the includes() method instead of a single character. If you use `numberArray.includes(char)` instead, it will pass the test!

Reply:
What does break do here?